modules/gitops-release-notes-1-11-3.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-11-3_{context}"]
+= Release Notes for {gitops-title} 1.11.3
+
+{gitops-title} 1.11.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-3_{context}"]
+== Errata updates
+
+[id="rhsa-2024-1697-gitops-1-11-3-security-update-advisory_{context}"]
+=== RHSA-2024-1697 - {gitops-title} 1.11.3 security update advisory
+
+Issued: 2024-04-08
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024-1697[RHSA-2024-1697]
+
+If you have installed the {gitops-title} Operator, view the container images in this release by running the following command:
```

Comment:
FYI - this line is phrased differently in other RN's in the section. Just a nit :)

Reply:
Fixed. Thanks for noticing.